htdocs/main.inc.php

```diff
@@ -1641,6 +1641,10 @@ function top_menu($head, $title = '', $target = '', $disablejs = 0, $disablehead
 				$text = '';
 				$title = $langs->trans($mode == 'wiki' ? 'GoToWikiHelpPage' : 'GoToHelpPage');
 				if ($mode == 'wiki') $title .= ' - '.$langs->trans("PageWiki").' &quot;'.dol_escape_htmltag(strtr($helppage, '_', ' ')).'&quot;'."";
+	            if (!empty($conf->global->MAIN_SHOWDATABASENAMEINHELPPAGESLINK)) {
```

Comment:
It seems this option is already managed few lines later (line 1658) toshow database name on the popup of version.